HISTORY.md

```diff
@@ -1,5 +1,7 @@
 ### mlpack ?.?.?
 ###### ????-??-??
+  * Added `mean squared logarithmic error` loss function (#2210).
```

Comment:
```suggestion
  * Added `mean squared logarithmic error` loss function for neural networks
    (#2210).
```

Just a quick fix; not everything in mlpack is neural networks, so it's nice to be specific. :)